tools/test.py

```diff
@@ -130,7 +129,14 @@ def main():
                                  args.gpu_collect)
 
     rank, _ = get_dist_info()
-    eval_cfg = cfg.get('evaluation', {})
+    eval_cfg = cfg.get('evaluation', args.metric_options)
+    # hard-code way to remove EvalHook args
```

Comment:
Any more elegant way to remove them?

Reply:
Remove it and use kwargs to receive EvalHook args.